sdk/python/feast/repo_config.py

```diff
@@ -52,6 +52,9 @@ class Config:
 class RegistryConfig(FeastBaseModel):
     """ Metadata Store Configuration. Configuration that relates to reading from and writing to the Feast registry."""
 
+    registry_store_provider: Optional[StrictStr]
```

Comment:
Reusing the name `provider` here is quite confusing since we have a `provider` concept already. Is it possible for us to infer the backend based on the URI?